components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/resources/store-api.yaml

```diff
@@ -234,6 +234,62 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
```

Comment:
Provide sample request and response using x-wso2-curl, x-wso2-request and x-wso2-response elements. Refer other resources in store-api.yaml

Reply:
https://github.com/wso2/carbon-apimgt/blob/v6.1.66/components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/resources/store-api.yaml#L469-L472